AirSkyBoat/AirSkyBoat
# Resolves reactor cool issue #904

… #904



**_I affirm:_**

## What does this pull request do?
Fixes typo that preventer Zdei type mobs from receiving Reactor Cool effect.

## Steps to test these changes
Apply fix. 
Go to Sea.
Engage a RDM or BLM Zdei.
Poke it until it uses reactor cool effect. 
Enjoy your spikes/paralyze debuff.

Note that the lua error messages now disappear from the server logs.
